clusterloader2/pkg/util/podstore.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
2018

Reply:
done.
I thought I should keep previous value because is mostly a copy.